docs/source/CONTRIBUTING.md

```diff
@@ -206,43 +219,55 @@ currently are:
   Bumping version from 2.0.1 to 2.1.0
   ```
 
-  - Note: All this is doing, is updating the version number in the
-    `pyproject.toml`. You can totally do this manually. This command just might
-    be a bit quicker. And it's nice to have a command to do it for you. Yay
-    automation!
+    - Note: All this is doing, is updating the version number in the
+      `pyproject.toml`. You can totally do this manually. This command just might be a bit quicker. And it's nice to
+      have a command to do it for you. Yay automation!
 - Update the CHANGELOG with the appropriate new version number and release date.
 - Create a pull request for these changes, and get it approved!
-- Once your PR has been merged, the final piece is to actually create the new
-  release.
+- Once your PR has been merged, the final piece is to actually create the new release.
 
-  1. Go to the `tartufo` [releases page] and click on `Draft a new release`.
-  2. Enter an appropriate tag version (in this example, `v2.1.0`).
-  3. Title the release. Generally these would just be in the form
-     `Version 2.1.0`. (Not very creative, I know. But predictable!)
-  4. Copy-paste the CHANGELOG entries for this new version into the description.
-  5. Click `Publish release`!
+    1. Go to the `tartufo` [releases page] and click on `Draft a new release`.
+    2. Enter an appropriate tag version (in this example, `v2.1.0`).
+    3. Title the release. Generally these would just be in the form
+       `Version 2.1.0`. (Not very creative, I know. But predictable!)
+    4. Copy-paste the CHANGELOG entries for this new version into the description.
+    5. Click `Publish release`!
 
-Congratulations, you've just issued a new release for `tartufo`. The automation
-will take care of the rest! 🎉
+Congratulations, you've just issued a new release for `tartufo`. The automation will take care of the rest! 🎉
 
 ## Additional Resources
 
 - [General GitHub Documentation](https://help.github.com/)
 - [GitHub Pull Request documentation](https://help.github.com/send-pull-requests/)
 
 [black]: https://github.com/psf/black
+
 [black-editors]: https://github.com/psf/black#editor-integration
+
 [bump the version]: https://python-poetry.org/docs/cli/#version
+
 [issues]: https://github.com/godaddy/tartufo/issues
+
 [Install Poetry]: https://python-poetry.org/docs/#installation
+
 [Poetry Environments]: https://python-poetry.org/docs/managing-environments/
+
 [our security policy]: https://github.com/godaddy/tartufo/security/policy
+
 [PEP 8]: https://www.python.org/dev/peps/pep-0008/
+
 [Poetry]: https://python-poetry.org/
+
 [poetry docs]: https://python-poetry.org/docs/
+
 [pre-commit]: https://pre-commit.com/
+
 [PyPI]: http://pypi.org/
+
 [releases page]: https://github.com/godaddy/tartufo/releases
+
 [slack]: https://godaddy-oss.slack.com/
+
 [tox]: https://tox.readthedocs.io/en/latest/
+
```

Comment:
Ummm... so, what's with all the extra lines down here? Do we really need these? 'cause they seem kinda... not good to me. 😄

Reply:
MarkDown auto-formatter is putting them in, I assumed they were coming from the .editorconfig

Reply:
Weird. Nope, editorconfig does no such thing. Sounds like a bug in that particular auto-formatter TBH. I'm definitely not a fan of that formatting choice since it breaks up what is a very intentional block. But doesn't have to be a hard blocker on getting this merged.

Reply:
Fixed